src/components/LoginButton.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+const LoginButtonStyled = styled.button`
+  width: 100px;
+  height: 30px;
+  border-radius: 10px 10px 10px 10px;
+  color: white;
+  opacity: 70%;
+  box-shadow: #e1e1e1 solid 3px;
+`;
+
+export default LoginButtonStyled;
```

Comment:
Make sure to name the component exactly like the file to have better auto complete support in VSCode.